bootstrapvz/providers/gce/tasks/image.py

```diff
@@ -56,6 +26,26 @@ def run(cls, info):
 		if 'description' in info.manifest.image:
 			image_description = info.manifest.image['description']
 		log_check_call(['gcutil', '--project=' + info.manifest.image['gce_project'],
-		                'addimage', info._gce['image_name'],
-		                info.manifest.image['gcs_destination'] + info._gce['tarball_name'],
+		                'addimage', info.image['name'],
+		                info.manifest.image['gcs_destination'] + info.image['tarball_name'],
```

Comment:
If the gce provider uploads the image itself, why does the user need to be able to specify the tarball name? If this were to run fully automatic, we could essentially delete the tarball after a successful run, right?